lms/templates/student_account/institution_register.underscore

```diff
@@ -0,0 +1,32 @@
+<div class="other-login-wrapper">
+    <div class="section-title lines">
+        <h2>
+            <span class="text">
+                <span class="icon fa fa-university" aria-hidden="true"></span>
+                <%- gettext("Register with Institution/Campus Credentials") %>
+            </span>
+        </h2>
+    </div>
+
+    <p><%- gettext("Choose your institution from the list below:") %></p>
+
+    <ul class="institution-list">
```

Comment:
I'm on the fence about this. There are subtle differences in the text and URLs used. I can combine them but it will have a number of interpolated `<% if %>` statements